driver/features.go

```diff
@@ -7,3 +7,5 @@ const DockerExporter Feature = "Docker exporter"
 
 const CacheExport Feature = "Cache export"
 const MultiPlatform Feature = "Multi-platform build"
+
+const DefaultLoad Feature = "Automatically load images to the Docker Engine image store"
```

Comment:
@dvdksn I think "Docker store" is enough no?

```suggestion
const DefaultLoad Feature = "Automatically load images to the Docker store"
```

Reply:
"Docker store" was the name of an old publisher tool of Docker Hub so that term becomes a little unfortunate for that reason. In general we always use "image store", and I think that keeping "Docker Engine" in there makes it clearer that we're referring to the local content store for the daemon.